tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ if (tensorflow_WIN_CPU_SIMD_OPTIONS)
     if(COMPILER_OPT_WIN_CPU_SIMD_SUPPORTED)
       set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${tensorflow_WIN_CPU_SIMD_OPTIONS}")
     else()
-      message("${tensorflow_WIN_CPU_SIMD_OPTIONS} not supported")
+      message(FATAL_ERROR "${tensorflow_WIN_CPU_SIMD_OPTIONS} not supported")
```

Comment:
I noticed if you you insert a wrong value as an unsupported instruction to this option when building it will always give you an error on further build attempts regardless if the argument was the supported instruction requiring to delete the build folder and starting again.